src/components/RHFInput.tsx

```diff
@@ -8,13 +10,23 @@ import {
   InputRightAddon,
   InputLeftAddon,
   Input,
+  StyleProps,
 } from '@chakra-ui/react';
-import PropTypes from 'prop-types';
 import get from 'lodash/get';
 
+type RHFInputProps = {
+    onChange?: ReactEventHandler,
+    onBlur?: ReactEventHandler,
+    name: string,
+    left?: ReactNode,
+    right?: ReactNode,
+    label: ReactNode,
+    errors?: Record<string, { message: string }>,
```

Comment:
I see most of these props on `RHFCheckBoxProps`, possible refactor is to find what is common, collect, import, and compose from there.